src/styles/index.scss

```diff
@@ -1,3 +1,62 @@
-body {
+@import './variables';
+
+.page {
   margin: 0;
+  padding: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  width: 100%;
+  max-width: 726px;
+  box-sizing: border-box;
+  padding: 10px;
+
+  &__space {
+    display: flex;
+    flex-wrap: wrap;
+    gap: $gap;
+  }
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: $square-size;
+    width: $square-size;
+    box-sizing: border-box;
+    border: 1px solid black;
+    background-color: $square-color;
+
+    &:hover {
+      cursor: pointer;
+      transform: translateY(-20px);
+      background-color: $square-color-hover;
+      animation-duration: 0.5s;
+    }
+  }
+}
+
+@for $number from 1 through 31 {
+  .calendar__day:nth-child(#{$number})::before {
+    content: '#{$number}';
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 30px;
```

Comment:
These styles are the same for all of `calendar__day`
Don't use loops for styles that stay the same
![image](https://github.com/user-attachments/assets/fa97465c-4c22-4c55-8855-01884e415362)